it/test/controllers/agent/manageBusinesses/cease/IncomeSourceEndDateControllerISpec.scala

```diff
@@ -55,7 +55,15 @@ class IncomeSourceEndDateControllerISpec extends ComponentSpecBase {
   val dateForeignPropertySubmitChangeAgentUrl: String = controllers.manageBusinesses.cease.routes.IncomeSourceEndDateController.submitChangeAgent(None, ForeignProperty).url
   val checkYourCeaseDetailsForeignPropertyShowAgentUrl: String = controllers.manageBusinesses.cease.routes.CeaseCheckIncomeSourceDetailsController.showAgent(ForeignProperty).url
 
-  val hintText: String = messagesAPI("dateForm.hint")
+  def hintText(incomeSourceType: IncomeSourceType): String = {
+    if (!(incomeSourceType == SelfEmployment)) {
```

Comment:
to improve readability, its worth to apply change here so we can avoid logical negation